src/api/app/views/webui/cloud/upload_jobs/new.html.haml

```diff
@@ -22,7 +22,7 @@
     = upload_job_form.label 'cloud_backend_upload_job[region]', "Region:"
     = select_tag('cloud_backend_upload_job[region]', options_for_select(@ec2_regions))
   %p
-    = upload_job_form.label 'cloud_backend_upload_job[vpc_subnet_id]', 'VPC Subnet ID (optional):'
+    = upload_job_form.label 'cloud_backend_upload_job[vpc_subnet_id]', 'VPC Subnet ID (eg. subnet-12345678, optional):'
```

Comment:
[nitpick] Maybe we can add another hint that we will create a temp VPC if the user does not provide one

Reply:
@ChrisBr I added a tooltip